README.rst

```diff
@@ -9,5 +9,4 @@ To run::
 	$ virtualenv -p python3 /var/tmp/deckhand
 	$ . /var/tmp/deckhand/bin/activate
 	$ sudo pip install .
```

Comment:
This comment is not directly related to this PR.  We should probably avoid using sudo for these commands,